commands/project/download.ts

```diff
@@ -83,10 +61,12 @@ exports.handler = async options => {
       }
     }
 
+    const absoluteDestPath = dest ? path.resolve(getCwd(), dest) : getCwd();
+
     const { data: zippedProject } = await downloadProject(
       derivedAccountId,
       projectName,
-      buildNumberToDownload
+      buildNumberToDownload!
```

Comment:
Are you sure we can assert that the build number exists? Haven't tested but I _think_ it could be `undefined` if you had project without any builds (is that still possible?)

Reply:
Good call. I was pretty sure that it would always exist (b/c otherwise something above would have thrown), but I didn't test that case. I'll try that

Reply:
Ok I just checked and you're right. I added an error message in [f82d7c8](https://github.com/HubSpot/hubspot-cli/pull/1412/commits/f82d7c89907fb62f41ed91af6d8953391bb79630)